R/recall_by_number.R

```diff
@@ -0,0 +1,33 @@
+#ua <- user_agent("http://github.com/WrathSmith/caRecall")
```

Comment:
what does this do in this context?

Reply:
It's from the **Set a user agent** section of https://httr.r-lib.org/articles/api-packages.html, not entirely sure if we need it, but it's suggested, it might be worth having as an option for a user to set, maybe leave it out for now